README.md

```diff
@@ -539,6 +539,104 @@ pathfinder:
           pathType: Prefix
 ```
 
+### Deploying Sombra on Azure
+
+The following example deploys Sombra on Azure. This would require setting up an Application Gateway, which will be used as the ingress controller, on Azure and adding the TLS secret into the cluster.
+
+```yaml
+imageCredentials:
+  registry: docker.transcend.io
+  username: Transcend
+  password: '<TRANSCEND_API_TOKEN>'
+
+transcend_service:
+  type: NodePort
+
+transcend_ingress:
+  enabled: true
+  className: 'azure-application-gateway'
+  annotations:
+    appgw.ingress.kubernetes.io/health-probe-hostname: <SOMBRA_TRANSCEND_INGRESS_DOMAIN>
+    appgw.ingress.kubernetes.io/health-probe-path: /health
+    appgw.ingress.kubernetes.io/health-probe-port: '5042'
+  hosts:
+    - host: <SOMBRA_TRANSCEND_INGRESS_DOMAIN>
+      paths:
+        - path: /
+          pathType: Prefix
+  tls:
+    - secretName: <SOMBRA_TRANSCEND_TLS_SECRET_NAME>
+      hosts:
+        - <SOMBRA_TRANSCEND_INGRESS_DOMAIN>
+
+customer_service:
+  type: NodePort
+
+customer_ingress:
+  enabled: true
+  className: 'azure-application-gateway'
+  annotations:
+    appgw.ingress.kubernetes.io/health-probe-path: /health
+    appgw.ingress.kubernetes.io/health-probe-port: '5039'
+  hosts:
+    - host: <SOMBRA_CUSTOMER_INGRESS_DOMAIN>
+      paths:
+        - path: /
+          pathType: Prefix
+  tls:
+    - secretName: <SOMBRA_CUSTOMER_TLSSECRET_NAME>
+      hosts:
+        - <SOMBRA_CUSTOMER_INGRESS_DOMAIN>
+
+envs:
+  - name: ORGANIZATION_URI
+    value: '<ORGANIZATION_URI>'
+  - name: EMPLOYEE_AUTHENTICATION_METHODS
+    value: 'transcend,session'
+  - name: DATA_SUBJECT_AUTHENTICATION_METHODS
+    value: 'transcend,session'
+  - name: TRANSCEND_URL
+    value: https://api.dev.trancsend.com
+  - name: SOMBRA_ID
+    value: '1af17328-e20c-4ba3-af63-f793cf1ecd13'
+  # [Optional] Add the following variables in order to utilize the AWS integrations
+  - name: AWS_ACCESS_KEY_ID
```

Comment:
Maybe just update the environment list as this example has nothing to do with AWS integrations

Reply:
this https://github.com/transcend-io/sombra-helm-chart/blob/7fe255c247e2a3d32b082e0638dd872a2faf19c4/README.md#configuring-sombra. You can leave the vars here too but also update the example description telling more about AWS integration.